js/map.mjs

```diff
@@ -363,7 +391,10 @@ async function initImgs(){
             "Nirnroot",
             "Check",
             "X",
-            "POI"
+            "POI",
+            "Fixed",
```

Comment:
maybe name them overlay_fixed, overlay_no_reroll, etc. to differentiate them from the default 40x40 icons?

Reply:
or icon_overlay_xxx

Reply:
Implemented